shade-test/jar-file/src/test/java/org/testcontainers/JarFileShadingTest.java

```diff
@@ -0,0 +1,79 @@
+package org.testcontainers;
+
+import org.assertj.core.api.ListAssert;
+import org.junit.*;
+
+import java.io.IOException;
+import java.net.URI;
+import java.nio.file.*;
+
+import static java.util.Collections.emptyMap;
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class JarFileShadingTest {
+
+    private static FileSystem fileSystem;
+
+    private static Path root;
+
+    @BeforeClass
+    public static void setUp() throws Exception {
+        Path path = Paths.get("..", "..", "core", "target", "testcontainers-0-SNAPSHOT.jar");
```

Comment:
Can you please elaborate why is it not IDE friendly?
I though one of Maven projects best practice is to avoid depending on explicit locations, but depends on artifacts. When you use copy dependencies you alleviate your self from that since you use GAV and not relative paths. You are free then to place core where ever you want.